perl/pushover.pl

```diff
@@ -78,7 +78,7 @@ sub init_config
 			weechat::config_set_plugin($option, $OPTIONS_DEFAULT{$option}[0]);
 			$OPTIONS{$option} = $OPTIONS_DEFAULT{$option}[0];
 		} else {
-			$OPTIONS{$option} = weechat::config_get_plugin($option);
+			$OPTIONS{$option} = weechat::string_eval_expression(weechat::config_get_plugin($option), {}, {}, {});
```

Comment:
You forgot the same bit for the "config_cb" subroutine, which is called whenever a config value is changed.

However I think it's even better to evaluate the expressions directly in the "notify" subroutine where they're being used, so I did that quickly myself in commit 1cb5b348e934a2665d6a28a79e6c2d28e9ae188a. :)